### PR TITLE
Update maintainers.md

### LINKED ISSUE #735: Update the Maintainers file

### PR DESCRIPTION
**Pull request by dharmjit**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** :
Fixes #735 

**Additional information**

**Special notes for your reviewer**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 | Maintainer       | GitHub ID                                              | Affiliation                              |
 |------------------|--------------------------------------------------------|------------------------------------------|
-| Anusha Hegde     | [anusha94](https://github.com/anusha94 )               | [VMware](https://www.github.com/vmware/) |
+| Shivani Singhal  | [shivi28](https://github.com/shivi28 )                 | [VMware](https://www.github.com/vmware/) |
```
Comment (anusha94) [inline MAINTAINERS.md:7]: Sort names alphabetically? That was the convention that we followed earlier.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 | Maintainer       | GitHub ID                                              | Affiliation                              |
 |------------------|--------------------------------------------------------|------------------------------------------|
-| Anusha Hegde     | [anusha94](https://github.com/anusha94 )               | [VMware](https://www.github.com/vmware/) |
```
Comment (anusha94) [inline MAINTAINERS.md:7]: Time to create a new list for Emeritus maintainers :)
Refer https://github.com/vmware-tanzu/velero/blob/main/MAINTAINERS.md#emeritus-maintainers

File: MAINTAINERS.md [hunk 3]
```diff
@@ -14,16 +14,9 @@
 |------|--------|------|
 | Tech Lead | Jamie Monserrate | [jamiemonserrate](https://github.com/jamiemonserrate) |
 | Tech Lead  | Shailesh Pant | [pshail](https://github.com/pshail) |
-| Eng Management | Gavin Lu |[luguang](https://github.com/luguang) |
 | Eng Management | Sudheesh Sudhakaran |[sudheesh-csp](https://github.com/sudheesh-csp) |
 | Eng Management | Vijayakumar Arumuga Nadar |[Vijayakumar0328](https://github.com/Vijayakumar0328) |
-| Product Management | Disha Chopra |[dishac](https://github.com/dishac) |
-| Engineering | Aleksandar Georgiev |[georgievaVMW](https://github.com/georgievaVMW) |
 | Engineering | Anusha Hegde |[anusha94](https://github.com/anusha94) |
```
Comment (anusha94) [inline MAINTAINERS.md:19]: I'd rather have my name removed from the engineering list as I won't be able to actively contribute anymore.

File: MAINTAINERS.md [hunk 4]
```diff
@@ -14,16 +14,9 @@
...
 | Engineering | Dharmjit Singh |[dharmjit](https://github.com/dharmjit/) |
-| Engineering | Hui Chen |[huchen2021](https://github.com/huchen2021) |
```
Comment (anusha94) [inline MAINTAINERS.md:24]: I am not sure of having an emeritus contributors list. But I think we can have one. BYOH wouldn't be what it is without the initial contributions of Hui, Yixing, Wuming, Rousko, and others :) 
I'd leave it to you though.

Comment (dharmjit) [reply MAINTAINERS.md:24]: I thought about this and it definitely makes sense to keep all the core contributors to date. I will take some hints from other OSS projects and any guidance I can get. Keeping a emeritus contributors list also seems like an option.

File: MAINTAINERS.md (hunk 2 above)
Comment (dharmjit) [reply MAINTAINERS.md:7]: Thanks for suggesting this.

**Comment by shivi28** [comment]
Thanks @dharmjit for these changes. 😄 
Special thanks to @anusha94 for all her contributions and guidance 🎉  🙏

**Review by shivi28 — APPROVED** [review]

**Review by Madhur97 — APPROVED** [review]

**Pull request merged**